src/XProfile/VirtoCommerce.XProfile/Commands/CreateOrganizationCommand.cs

```diff
@@ -13,7 +13,6 @@ public CreateOrganizationCommand(string name, IList<Address> addresses)
         }
 
         public string Name { get; set; }
-        public IList<Address> Addresses { get; set; } = new List<Address>();
-        
+        public IList<Address> Addresses { get; set; }

```

Comment:
Why did you remove the default value?

Reply:
Member initializer values should not be redundant: https://rules.sonarsource.com/csharp/RSPEC-3604